botfront/cypress/support/index.js

```diff
@@ -74,20 +74,44 @@ Cypress.Commands.add('createNLUModel', (projectId, name, language, description,
     cy.get('[data-cy=model-save-button]').click();
 });
 
-Cypress.Commands.add('createNLUModelWithImport', (projectId, name, language, description, instance) => {
-    cy.createNLUModel(projectId, name, language, description, instance);
-    cy.fixture('bf_project_id.txt').then((id) => {
-        cy.visit(`/project/${id}/nlu/models`);
-    });
-    cy.contains('French').click();
-    cy.get(':nth-child(1) > .extra > .basic > .primary').click();
-    cy.get('.nlu-menu-settings').click();
-    cy.contains('Import').click();
-    cy.fixture('nlu_import.json', 'utf8').then((content) => {
-        cy.get('.file-dropzone').upload(content, 'data.json');
-    });
+Cypress.Commands.add('createNLUModelProgramatically', (projectId, name, language, description) => {
+    cy.MeteorCall('nlu.insert', [
+        {
+            name: `${name}`,
+            language: `${language}`,
+            description: `${description}`,
+        },
+        `${projectId}`,
+    ]);
+});
+
+Cypress.Commands.add('MeteorCall', (method, args) => {
+    cy.window().then(
+        ({ Meteor }) => new Cypress.Promise((resolve) => {
+            Meteor.call(method, ...args, (err, res) => {
+                if (err) resolve(err);
+                resolve(res);
+            });
+        }),
+    );
+});
 
-    cy.contains('Import Training Data').click();
+Cypress.Commands.add('createNLUModelWithImport', (projectId, name, language, description) => {
+    let modelId = '';
+    cy.createNLUModelProgramatically(projectId, name, language, description)
+        .then((result) => {
+            modelId = result;
+            cy.fixture('bf_project_id.txt').then((id) => {
+                cy.visit(`/project/${id}/nlu/model/${modelId}`);
+            });
+            cy.get('.nlu-menu-settings').click();
+            cy.contains('Import').click();
+            cy.fixture('nlu_import.json', 'utf8').then((content) => {
+                cy.get('.file-dropzone').upload(content, 'data.json');
+            });
+        
+            cy.contains('Import Training Data').click();
+        });
```

Comment:
If I read well, you created a command to create a model from a JSON file, but remove the UI test for imports. We should still verify that importing data works. (see comment above)

Reply:
Ok then I will add it again.